pyramid_swagger/tween.py

```diff
@@ -248,6 +244,12 @@ def files(self):
     def json(self, **kwargs):
         return getattr(self.request, 'json_body', {})
 
+    def __getattr__(self, name):
```

Comment:
Exactly.  The test includes hitting the `body` attribute to make sure that it's still calling the right property method, not hitting the fallback.